assets/js/common.js

```diff
@@ -762,6 +762,29 @@ var hivepress = {
 				});
 		});
 
+		// Number
+		container.find('input[type="number"]').each(function () {
```

Comment:
Please target only our own number fields to avoid affecting other plugin fields (e.g. WooCommerce ones). If our number fields have no specific data-component attribute already, you can add data-component="number" in the Number field class rendering.